rs/execution_environment/src/execution/response/tests.rs

```diff
@@ -2657,7 +2658,7 @@ fn cycles_balance_changes_applied_correctly() {
 
     let mut b = wasm().accept_cycles(Cycles::new(u128::MAX));
 
-    for _ in 0..400 {
+    for _ in 0..4 {
```

Comment:
Not sure why the test made so many parallel calls so I reduced their number to stay above the freezing threshold.